azavea/opendataphilly-jkan
# Facilitate imports from CKAN site (#301)

Facilitate importing datasets, organizations, and organization logos from the existing production CKAN site.

Potential improvements
- the whole script could be in Python for easier maintenance
- the import script currently redundantly downloads organization logos, which makes it take a little longer

Testing instructions:
- Follow the import/README steps and confirm that they make sense
- At this point they shouldn't result in any new changes unless prod ODP gets updated, because the imported data is all checked in to git

Resolves [#301](https://github.com/azavea/geospatial-apps-tasks/issues/301)
